packages/main/src/themes/TabContainer.css

```diff
@@ -77,10 +77,15 @@
 	margin-top: 0.25rem;
 	border-radius: 0.75rem;
 	height: 1.5rem;
+	border: 0.25rem solid transparent; /* to reserve space for the outside border and prevent flickering*/
 }
 
 .ui5-tc__overflow > [ui5-button][focused] {
 	outline-offset: 0.125rem;
+	--_ui5_button_focused_border_radius: 0.75rem;
+	--_ui5_button_focused_border: none;
+	padding: 0.125rem;
+	border: 2px solid var(--sapContent_FocusColor);
```

Comment:
change the `border` to `outline`, use rem values
remove the padding from the above line and remove the border from line 80

the result should be a focus that doesn't make the button grow in size

Reply:
Done